lua/cfc_time/server/storage_options/mysql.lua

```diff
@@ -40,6 +68,63 @@ function storage:InitQuery( rawQuery )
     return query
 end
 
+function storage:GetMaxSessionTime( callback )
+    local queryStr = [[
+        SELECT COLUMN_TYPE
+        FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE TABLE_NAME = "sessions"
+        AND COLUMN_NAME = "duration"
+    ]]
+
+    local query = self:InitQuery( queryStr )
+
+    local maxSessionTime = function( data )
+        -- e.g.
+        -- "mediumint(8)"
+        -- "mediumint(8) unsigned"
+        local columnData = data[1].COLUMN_TYPE
+
+        -- e.g.
+        -- { [1] = "mediumint(8)" }
+        -- { [1] = "mediumint(8)", [2] = "unsigned" }
+        local spl = string.Split( columnData, " " )
+
+        -- e.g.
+        -- "mediumint"
+        -- "mediumint"
+        local column = string.Split( spl[1], "(" )[1]
+
+        -- e.g.
+        -- true
+        -- false
+        local signed = spl[2] ~= "unsigned"
+
+        logger:debug( "Getting max session duration for column: [" .. column .. "] (signed: " .. tostring( signed ) .. ")" )
+        return MAX_SESSION_DURATIONS[column][signed and "signed" or "unsigned"]
+    end
+
+    query.onSucces = function( _, data )
+        if data then
+            logger:debug( "Session duration query result", data )
+        end
+
+        local maxTime = maxSessionTime( data )
+
+        callback( maxTime )
+    end
+
+    query:start()
+end
+
+function storage:SetMaxSessionTime()
```

Comment:
Seems weird to have a function starting with "set" that takes no args, perhaps it should be like `CacheMaxSessionTime()`?